buildingmotif-app/src/app/pointlabel-parser/pointlabel-parser.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { HttpErrorResponse, HttpHeaders } from '@angular/common/http';
+import { Model } from '../types'
+import { Observable, throwError } from 'rxjs';
+import { catchError, retry } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PointlabelParserService {
+
+  constructor(private http: HttpClient) { }
+
+  parse(pointlabels: string, parsers: string): Observable<Model | any> {
+    const headers = {'Content-Type': "application/json"}
+
+    return this.http.post(`http://localhost:5000/parsers`, {point_labels: pointlabels, parsers}, {headers, responseType: 'json'})
```

Comment:
I know we've talked about this before, and it's not specific to this PR, but the base URL for the api requests should be broken out into an environment variable.